### PR TITLE
Add support for community repositories

### PR DESCRIPTION
**Pull request by MagneFire**
This PR adds the ability for other layers to define a set of packages to install in the package feed for use with `opkg`. An example use is like what I've done in the `meta-games` repo: https://github.com/MagneFire/meta-games/blob/community_repo/conf/layer.conf#L13.

A nice thing is that packages won't be build in the package feed when a layer doesn't exist. So it decouples the community packages from the main repos.

Also, because of this added functionality I have removed the creation of the package feed from the `bitbake asteroid-image` command.
So users building AsteroidOS don't actually build the package feed. This however does mean that an adjustment has to be made on the build server such that after a asteroid-image build the `bitbake asteroid-package-feed` command is used to update the package feed and with it the community packages.

### CODE REVIEW

File: recipes-asteroid/asteroid-package-feed/asteroid-package-feed.bb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+SUMMARY = "Build community packages and create package index from the community packages and rootfs pacakges."
+LICENSE = "MIT"
+
+INHIBIT_DEFAULT_DEPS = "1"
+PACKAGES = ""
+
+# A set of packags to build for the package feed bu not include in rootfs.
```
Comment (FlorentRevest) [inline recipes-asteroid/asteroid-package-feed/asteroid-package-feed.bb:7]: but that are not included*

File: recipes-asteroid/asteroid-package-feed/asteroid-package-feed.bb [hunk 2]
```diff
@@ -0,0 +1,30 @@
+SUMMARY = "Build community packages and create package index from the community packages and rootfs pacakges."
```
Comment (FlorentRevest) [inline recipes-asteroid/asteroid-package-feed/asteroid-package-feed.bb:1]: I would reword that as "Create a package index from the packages included in rootfs and community packages"

**Review by FlorentRevest — APPROVED** [review]

File: recipes-asteroid/asteroid-package-feed/asteroid-package-feed.bb (hunk 1 above)
Comment (MagneFire) [reply recipes-asteroid/asteroid-package-feed/asteroid-package-feed.bb:7]: ow, wow. Looks almost as if my keyboard is broken :smile:
Sorry for not checking this...

**Review by jrtberlin — APPROVED** [review]
